packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -406,7 +407,12 @@ export const getRoutes = (locals) => {
     if (typeof routes === 'function') {
         _routes = routes()
     }
-    const allRoutes = [..._routes, {path: '*', component: Throw404}]
+    const allRoutes = [
+        // NOTE: this route needs to be above _routes, in case _routes has a fallback route of `path: '*'`
+        {path: '/__mrt/refresh', component: RefetchData},
```

Comment:
Should `/__mrt/refresh` be below the users defined routes so they cannot override it?

Reply:
Routes are matched in the order they are present in the array, so it should go before user-defined routes to prevent overrides. (Which is also why our default 404 component is _last_, so that it's only used as a last resort.)

Previous discussion: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1488#discussion_r1362487227